contracts/src/LightClientArbitrum.sol

```diff
@@ -9,6 +9,11 @@ interface ArbSys {
 }
 
 contract LightClientArbitrum is LightClient {
+    /// @custom:oz-upgrades-unsafe-allow constructor
+    constructor() {
```

Comment:
why do we need to do this? it should inherit the constructor from LightClient

Reply:
that's right, this code lingered here because i originally planned to override the initialize function and wanted to be explicit. will remove